.github/actions/build-refs/action/index.ts

```diff
@@ -25,6 +20,23 @@ export type AttemptableRefs = Map<Repo, AttemptableRef[]>
 
 export type OutputRefs = Map<Repo, Ref>
 
+export function variantForRef(ref: Ref): Variant {
+  if (ref.startsWith('refs/heads')) {
+    if (ref.includes('internal-release')) {
+      return 'internal-release'
+    } else {
+      return 'release'
+    }
+  } else if (ref.startsWith('refs/tags')) {
+    if (ref.includes('ot3@')) {
```

Comment:
is this tag valid for buildroot?

Reply:
sadly, yes, that's the internal-release tag. We need to change it someday.